.changeset/great-guests-collect.md

```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/identity-apps-core": major
+"@wso2is/console": minor
```

Comment:
ned to bump the features as well

Reply:
Changed apps/modules are - identity-apps-core, features, and i18n. Therefore let's update the versions of those packages.

Also, we are not introducing a breaking change to identity-apps-core, let's not bump it to a major version.

Since we are adding a new feature, we can bump features and identity-apps-core to a minor version, and i18n to a patch version.

Reply:
Still the i18n module is not bumped to a patch version.

Also, what is the reason for updating identity-apps-core to a major version? since this is a feature addition, shouldn't it be `minor`?

cc: @brionmario

Reply:
Changed with https://github.com/wso2/identity-apps/pull/5817/commits/186c8e7bf9640ef6cbb0a56d4f12dd881355e639

i18n set to minor since new introductions are there.